salesagility/SuiteCRM
# Feature: Allow configuring the Calendar name for the Google Sync via config

## Description




This is a small change to allow configuring the Calendar name used when syncing to Google.
## Motivation and Context

Calendar names may want to be changed for aesthetic, branding or business purposes. This change provides the smallest change to implement this.
Future improvements may allow configuring this via admin or on a per-user basis.

## How To Test This

Using an instance configured with the Google Sync set the config value `google_calendar_sync_name` to anything other than `SuiteCRM`. Sync the calendars and confirm that your Google calendar has a new calendar with the name corresponding to the setting with the CRM meetings.

## Types of changes


### Final checklist
